encrypt.go

```diff
@@ -66,6 +67,24 @@ var ContentEncryptionAlgorithm = EncryptionAlgorithmDESCBC
 // content with an unsupported algorithm.
 var ErrUnsupportedEncryptionAlgorithm = errors.New("pkcs7: cannot encrypt content: only DES-CBC, AES-CBC, and AES-GCM supported")
 
+// KeyEncryptionAlgorithm determines the algorithm used to encrypt a
+// content key. Change the value of this variable to change which
+// algorithm is used in the Encrypt() function.
+var KeyEncryptionAlgorithm = OIDEncryptionAlgorithmRSA
```

Comment:
If possible, we should use `RSA OAEP` by default.

Reply:
Are we considering adding new encrypt methods so we don't have to change the value of this variable? 

The same applies to `ContentEncryptionAlgorithm`. And should we default to `EncryptionAlgorithmAES256GCM`. I guess we cannot use `XSalsa20+Poly1305` here.

Reply:
Right now it uses the current default, making this fork backwards compatible with the current Mozilla one (with some fixes and new features). But I agree that defaulting to RSA OAEP is better going forward. Same applies to `EncryptionAlgorithmAES256GCM`.

For `XSalsa20+Poly1305` in CMS there exists a proposed standard: https://datatracker.ietf.org/doc/html/rfc8103. So I guess it can be added; it's just a bit less "standard" than the other algorithms, but still something that has been reviewed by IETF.

Yes, changing the API of this package is something I would like to do. If we add new functions, the changes _could_ be backwards compatible. But it might also make sense to start a new major version, so that there's no need to be fully backwards compatible.